src/dropdown.ts

```diff
@@ -318,15 +318,15 @@ export default class Dropdown extends LitElement implements FormControl {
     }
 
```

Comment:
We've now had two bugs where "Select All" found its way into either Dropdown's internal label or its input field. That made me realize there's a structural issue that needed to be addressed. I think I addressed it below by simplifying things quite a bit:

TODO